Loop/Managers/DeviceAlert/InAppModalDeviceAlertHandler.swift

```diff
@@ -72,17 +76,18 @@ extension InAppUserAlertHandler {
         }
     }
     
-    private func show(alert: UserAlert) {
+    private func show(alert: DeviceAlert) {
         guard let content = alert.foregroundContent else {
             return
         }
         DispatchQueue.main.async {
             guard self.alertsShowing.contains(where: { $1.identifier == alert.identifier }) == false else {
                 return
             }
-            let alertController = self.presentAlert(title: content.title, message: content.body, action: content.acknowledgeAction) {
+            let alertController = self.presentAlert(title: content.title, message: content.body, action: content.acknowledgeActionButtonLabel) {
                 self.alertsShowing.removeAll { $1.identifier == alert.identifier }
-                alert.acknowledgeCompletion?(alert.typeIdentifier)
```

Comment:
Not really, because I don't want the other side-effects from `removeDeliveredAlerts` (in this case, I don't need to call `dismiss` in the dismiss-completion callback here...)